nservicebus/pipeline/steps-stages-connectors_extra_core_[6.0,).partial.md

```diff
@@ -14,6 +14,7 @@ In the diagram User Code can refer to a handler or a saga. If the handler or sag
 
 ### Incoming Pipeline Stages
 
+ * TransportReceiveContext: Behaviors in this stage are responsible signaling failure or success to the transport. This stage provides `ITransportReceiveContext` to it's behaviors.
```

Comment:
Behaviors in this stage are responsible signalling failure or success to the transport. In this stage no outgoing operations are supported. This stage provides `ITransportReceiveContext` to its behaviors.